src/reverse-bits.js

```diff
@@ -0,0 +1,37 @@
+// Taken from bit-twiddle library
```

Comment:
Might as well just add this as a dependency

Reply:
I can do that, but it doesn't implement reverseBits16, which we need. We'd end up having the same code in two places. Specifically the lookup table for reversing bytes.

Reply:
I see. No worries, stick with this for now.